05/app.js

```diff
@@ -0,0 +1,56 @@
+const form = document.querySelector('form');
+form.noValidate = true;
+const messages = document.querySelector('.messages');
+const btn = document.querySelector('[type="submit"]');
+const required = form.querySelectorAll("[required]");
+const postCode = form.querySelector('[name="zip"]');
+let errors = [];
+
+form.addEventListener('submit', function(el){
```

Comment:
Tutaj powinno być nie `el`, a `event` albo `e` - inaczej wprowadzasz w błąd :) przecież to nie element